src/softmax.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+import {div, sub} from './binary.js';
+import {exp} from './unary.js';
+import {reduceMax, reduceSum} from './reduce.js';
+
+/**
+ * Compute the softmax values of the 2-D input tensor along axis 1.
+ * @param {Tensor} x
+ * @return {Tensor}
+ */
+export function softmax(x) {
+  if (x.rank !== 2) {
+    throw new Error('The input is not a 2-D tensor.');
+  }
+  const maxX = reduceMax(x, {axes: [1], keepDimensions: true});
```

Comment:
There's a typos on https://github.com/webmachinelearning/webnn/blob/main/index.bs#L1901
```
    const max_x = builder.reduceMax(x, { axes: [1], keepDimensions: true });
    const exp_x = builder.exp(builder.sub(x, max));  // L1901
```
it needs update to
```
    const exp_x = builder.exp(builder.sub(x, max_x));
```

@huningxin PTAL, thanks.

Reply:
Yes, that's one. Could you please file an issue to spec?

Reply:
Done, https://github.com/webmachinelearning/webnn/issues/246

Reply:
thanks!